cl/donate/models.py

```diff
@@ -164,3 +166,85 @@ def __str__(self) -> str:
             self.monthly_donation_amount,
             self.get_payment_provider_display(),
         )
+
+
+class NeonWebhookEvents(models.Model):
+    MEMBERSHIP_CREATION = 1
+    MEMBERSHIP_EDIT = 2
+    MEMBERSHIP_DELETE = 3
+    MEMBERSHIP_UPDATE = 4
+    TYPES = (
+        (MEMBERSHIP_CREATION, "createMembership"),
+        (MEMBERSHIP_EDIT, "editMembership"),
+        (MEMBERSHIP_DELETE, "deleteMembership"),
+        (MEMBERSHIP_UPDATE, "updateMembership"),
+    )
+    trigger = models.PositiveSmallIntegerField(
+        help_text="The current membership tier of a user within Neon CRM",
+        choices=TYPES,
+        null=True,
+    )
+    account_id = models.CharField(
+        help_text="Unique identifier assigned by Neon CRM to a customer record",
+        default="",
```

Comment:
I don't think you need to set the default to blank if you've got `blank=True`, do you?

Reply:
You're right. I'm removing the `default` argument.